cross/i2pd/PLIST

```diff
@@ -0,0 +1,2 @@
+bin:bin/i2pd
+lib:lib/libi2pd.a
```

Comment:
I do not expect a static archive library to be of any use in DSM system when no compiler is available. Please remove it.